src/ui/components/results/universalresultscomponent.js

```diff
@@ -85,7 +88,10 @@ export default class UniversalResultsComponent extends Component {
       // Show a view more link by default, which also links to verticalURL.
       viewMore: true,
       // By default, the view more link has a label of 'View More'.
-      viewMoreLabel: defaultConfigOption(config, ['viewMoreLabel', 'viewAllText'], 'View More'),
+      viewMoreLabel: defaultConfigOption(config, ['viewMoreLabel', 'viewAllText'], TranslationFlagger.flag({
```

Comment:
LL. Let's also put `config` and the array on their own lines. That way, all three args start on a newline.

Reply:
Updating! What is our line length for this repo? This one flew under my radar since it's significantly shorter than lines ~104-123 of this file.